saltgui/index.html

```diff
@@ -27,9 +27,32 @@ <h1>SaltGUI</h1>
           </div>
           <input id='username' type='text' placeholder='Username' />
           <input id='password' type='password' placeholder='Password' />
+          <select id='eauth'>
+            <!-- see https://docs.saltstack.com/en/latest/ref/auth/all/index.html -->
+            <option id='eauth_default' value="default">Type</option>
+            <optgroup label="standard" />
+              <option value="pam">pam</option>
+            </optgroup>
+            <optgroup label="supported" />
+              <!-- move items to this optgroup when at least -->
+              <!-- one user reports a succesful use -->
+              <option value="auto">auto</option>
+              <option value="file">file</option>
+              <option value="sharedsecret">sharedsecret</option>
+              <option value="yubico">yubico</option>
+            </optgroup>
+            <optgroup label="experimental" />
+              <option value="django">django</option>
+              <option value="keystone">keystone</option>
+              <option value="ldap">ldap</option>
+              <option value="mysql">mysql</option>
+              <option value="pki">pki</option>
+              <option value="rest">rest</option>
```

Comment:
Experimental means untested / assumptions. I prefer not to have this in the codebase: just add new login types when they are fully tested or are a requirement. So we should add `ldap` as well and check with @babs if this works.

Reply:
agreed, done